pepper/Makefile

```diff
@@ -43,15 +43,20 @@ link-exo: check-env
 	ln -sf ~/dex-zksnarks/exo_compute/private_input.py $(PEPPER_BIN)/exo0
 
 # Target to run all tests (e.g. used by travis)
-test: unit-tests gadget-tests e2e-tests
+test: unit-tests-iBV unit-tests gadget-tests e2e-tests
 
 e2e-tests: check-env
 	test/e2e/hash_transform.sh
 
 unit-tests: test/apps/hash_transform_test.cpp
-	$(CXX) -DTEST $(TEST_IFLAGS) $< -o $(TEST_BIN)/run_unit_tests $(TEST_LDFLAGS)
+	$(CXX) $(CXXFLAGS) -DTEST $(TEST_IFLAGS) $(IFLAGS) $< -o $(TEST_BIN)/run_unit_tests $(TEST_LDFLAGS) $(LDFLAGS) 
 	$(TEST_BIN)/run_unit_tests
 
+unit-tests-iBV: test/apps/isBoolVerification_test.cpp
```

Comment:
let's please not use abbreviations like this. It might save you 15 characters typing now, but it will make it harder to see what this is testing without also finding the test file name (which if we start like this might become abbreviated as well).

Reply:
you are right